### DISCUSSION TITLE
Can cassandra database be optional for knowlg APIs?

### DISCUSSION THREAD

**Discussion by vaivk369** [Sunbird-Knowlg/Community#204] [General Discussions]
@pallakartheekreddy  @AmiableAnil
We have planned to use Sunbird Knowlg to create and consume contents, collections for one of the clients. We got to know that knowlg team is working to make storing data into cassandra as an optional thing. 
Wanted to know when is this actually planned?

**Comment by pallakartheekreddy** [comment]
@vaivk369 
Please find below the details regarding making the Cassandra database optional for Knowledge Services:

1. Taxonomy Service: The Cassandra database has already been made optional for the Taxonomy service.
2. Content Service: This item is currently in the backlog, and we plan to address it in an upcoming release. We will provide updates on this thread once we have prioritized this item.

**Reply by pallakartheekreddy** [reply]
@vaivk369 Pls let us know if your query is resolved
